core/templates/dev/head/services/explorationServices.js

```diff
@@ -19,7 +19,7 @@
 
 // A service that provides a number of utility functions useful to both the
 // editor and player.
-oppia.factory('oppiaExplorationHtmlFormatterService', [
+oppia.factory('ExplorationHtmlFormatterService', [
   '$filter', 'extensionTagAssemblerService', 'HtmlEscaperService',
   function($filter, extensionTagAssemblerService, HtmlEscaperService) {
     return {
```

Comment:
And this too, let's rename it from `explorationServices.js` to `ExplorationHtmlFormatterService`.